docs/interactive/launchbuttons.ipynb

```diff
@@ -19,7 +19,24 @@
     "  url                       : https://github.com/yourusername/yourbookrepo  # Online location of your book\n",
     "  path_to_book              : path/to/book  # Optional path to your book, relative to the repository root\n",
     "  branch                    : master  # Which branch of the repository should be used when creating links (optional)\n",
-    "```"
+    "```\n",
+    "The launch button will use [nbgitpuller](https://github.com/jupyterhub/nbgitpuller) to pull your\n",
```

Comment:
This isn't _quite_ true - JupyterHub will use nbgitpuller, but Colab and Binder use their own mechanisms for pulling in content. Could We just remove the mention of nbgitpuller and start with `The repository:`?

Reply:
Thanks for clarifying. 

If users are trying to troubleshoot, what are the mechanisms for pulling the notebook into 
* Colab = 
* Binder = 
* JupyterHub = nbgitpuller

These details might not belong here in this explanation, but it might be nice to know how the notebooks appear in the environments.

Reply:
Binder uses a tool called repo2docker for this: https://repo2docker.readthedocs.io/

while I'm not sure what Colab uses since it's all proprietary